### DISCUSSION TITLE
AttributeError: module 'numpy' has no attribute 'float'.

### DISCUSSION THREAD

**Discussion by roseyunzi** [roboflow/supervision#912] [Q&A]
### Search before asking

- [x] I have searched the Supervision [issues](https://github.com/roboflow/supervision/issues) and found no similar feature requests.

### Question

How do i solve this problem?
when i Predict and annotate whole video
it raised error "AttributeError: module 'numpy' has no attribute 'float'."
![image](https://github.com/roboflow/supervision/assets/76989423/92bbecf9-9438-4081-a574-f8c35de36906)
![image](https://github.com/roboflow/supervision/assets/76989423/8cdf2d5e-7fa0-4e93-b32f-82ff70cf02aa)

![image](https://github.com/roboflow/supervision/assets/76989423/c89aeb85-e30f-4640-a43c-9a8ffc495951)

**Comment by SkalskiP** [comment]
Hi @roseyunzi 👋🏻 Thanks for your interest in `supervision`. What version of `numpy` you have installed?

**Comment by SkalskiP** [comment]
I'll convert this issue into a discussion and put it in the QA section.

**Comment by nextgenai-research** [comment]
Hi @SkalskiP, I am also facing the same issue. Can you please help?

AttributeError: module 'numpy' has no attribute 'float'.
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
[<ipython-input-18-faff2fc32589>](https://localhost:8080/#) in <cell line: 17>()
     29         detections.filter(mask=mask, inplace=True)
     30         # tracking detections
---> 31         tracks = byte_tracker.update(
     32             output_results=detections2boxes(detections=detections),
     33             img_info=frame.shape,

3 frames
[/usr/local/lib/python3.10/dist-packages/numpy/__init__.py](https://localhost:8080/#) in __getattr__(attr)
    317 
    318         if attr in __former_attrs__:
--> 319             raise AttributeError(__former_attrs__[attr])
    320 
    321         if attr == 'testing':

AttributeError: module 'numpy' has no attribute 'float'.
`np.float` was a deprecated alias for the builtin `float`. To avoid this error in existing code, use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here.
The aliases was originally deprecated in NumPy 1.20; for more details and guidance see the original release note at:
    https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations

**Reply by nextgenai-research** [reply]
Hello @roseyunzi  - I fixed the issue. Just update below code in Install ByteTrack

!pip install "numpy<1.24"

![Screen Shot 2024-04-03 at 12 59 44 PM](https://github.com/roboflow/supervision/assets/165880161/91bf2afd-a340-42cd-9318-a59d670cc3eb)
